src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int resultColor = random.nextInt(DifferentColors.values().length);
+        return String.valueOf(DifferentColors.values()[resultColor]);
```

Comment:
as described in common mistakes you should use name() method instead of toString to get string representation of enum